def error():
    raise Exception("Pyc exception")
